app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -1032,7 +1031,7 @@ class ChatRoomPresenter @Inject constructor(
                         ChatRoom(
                             id = id,
                             subscriptionId = subscriptionId,
-                            parentId = parentId,
+                            //parentId = parentId,
```

Comment:
? It won't compile. Why commenting this needed parameter?

Reply:
I was receiving errors for the parentID reference. Peter was as well while he was using JDK 11 but using JDK 10 fixed it for him. I followed the Rocket.Chat readme instructing us to use Java 8 and compiled my project with jdk 1.8.0_172 accordingly. I'm still seeing the errors though so this was a temporary fix for me. I don't intend on pushing this change to Rocket.Chat repository. If you have any idea why I am getting an error on this reference, do let me know.